EarlGrey/Action/GREYActions.m

```diff
@@ -415,7 +421,13 @@ + (void)grey_setText:(NSString *)text onWebElement:(id)element {
                                           object:element];
         [NSNotificationCenter.defaultCenter postNotification:notification];
       }
-
+      if (elementIsUITextView) {
```

Comment:
Why not combine this with elementIsUITextField?
```
if (elementIsUITextField || elementIsUITextView) {
  NSString *notificationName = elementIsUITextField ? UITextFieldTextDidChangeNotification : UITextViewTextDidChangeNotification;
  // Send notification with notification name.
}
```

Reply:
because this way is much more clear and readable?

Reply:
I'd have preferred clubbing them since on looking at the code blocks, it took a bit to find out that the only difference was "View" and "Field" between them. That's fine though, this was more of a nit.

Reply:
Reason why I don't like that:

It roughly translates to below after you unroll the ternary operator:

if (elementIsUITextField || elementIsUITextView) {
  NSString *notifcationName;
  if (elementIsUITextField) {
    notifcationName = UITextFieldTextDidChangeNotification;
  } else {
    notifcationName = UITextViewTextDidChangeNotification;
  }
  NSNotification *notification =
            [NSNotification notificationWithName:notifcationName
                                                                object:element];
  [NSNotificationCenter.defaultCenter postNotification:notification];
}

when things can be clearly expressed with non-nested ifs, it's less painful to my eyes and for my brain.

Reply:
It's so great.